cmd/kured/main.go

```diff
@@ -278,6 +281,9 @@ func main() {
 	go rebootAsRequired(nodeID, rebooter, rebootChecker, window, lock, client)
 	go maintainRebootRequiredMetric(nodeID, rebootChecker)
 
+	if pprofEnabled {
```

Comment:
I don't think this works: while pprofEnabled is by defaut to false, the net/http/pprof is still loaded, so it's still injecting itself?

Reply:
Oh  yeah.. :facepalm: 

i guess we could just `import _ "net/http/pprof"` and leave it enabled. 
Or use a custom mux and not the DefaultServeMux where we can conditionally enable it. 

Which do you like?

Reply:
I don't know yet... Do we really need to have this enabled at all times? For me we do not... And we do not have to flip the switch live on a build. I don't mind continuing to add `import _ "net/http/pprof"` line when necessary.

For me the valid question is not about whether we do have profiling, but rather "who is checking the profiling before release" :)

Reply:
let's just add `import _ "net/http/pprof"` then?

Reply:
Yeah that's why I would like it default off but on for all our hundreds of instances.